spaces/daniell-mesquita/skin.scss

```
@@ -0,0 +1,13 @@
+.daniell-mesquita {
+  --primary-color: #f74174;
+--bg-color: #f74174;
+--text-color: #fff;
+--link-color: #fff;
+--heading-color: #fff;
+--border-color: #04b7bc;
+--header-bg: #04b7bc;
+--block-bg: transparent;
+  }
+}
```

Scss indentation is not correct, you have to delete the extra quote. "}"
```suggestion
.daniell-mesquita {
  --primary-color: #f74174;
  --bg-color: #f74174;
  --text-color: #fff;
  --link-color: #fff;
  --heading-color: #fff;
  --border-color: #04b7bc;
  --header-bg: #04b7bc;
  --block-bg: transparent;
}
```